src/modules/dash/components/detailModal.vue

```diff
@@ -166,10 +196,25 @@ const editInfos = ref(false);
         type="warning"
         title="Cuidado..."
         text="Modo de edição ativo!"
-        closable=""
         v-if="editInfos"
       >
       </v-alert>
+
+      <Confirmation
+        @close="confirmation = false"
+        @delete-item="
+          this.$emit('delete-item', info.id);
+          confirmation = false;
+        "
+        @update-item="
+          this.$emit('update-item', info);
```

Comment:
no template é possível utilizar sem o this
```suggestion
          $emit('update-item', info);
```